### ISSUE TITLE
Windows 10, Code View, Pressing <ctrl>+<home> gives error message

### ISSUE THREAD

**Issue by tpronk** [psychopy/psychopy#3251]
I did the following:
- Create a new file in code view
- Press `<ctrl>+<home>`

I then get the following error message:
```
Traceback (most recent call last):
  File "C:\Program Files\PsychoPy3\lib\site-packages\psychopy\app\coder\coder.py", line 2594, in foldAll
    self.currentDoc.FoldAll()
TypeError: StyledTextCtrl.FoldAll(): not enough arguments
```

**Comment by peircej** [comment]
Most likely this is something that's changed at some point in wxpython (and we don't make a call to that function in our test suite). I don't use the function enough to have noticed

**Comment by TEParsons** [comment]
fixed in #6d4f66b0
